generic-concepts.ttl

```diff
@@ -16,7 +16,7 @@
 
 Released under the Creative Commons license Attribution 4.0 International (CC BY 4.0)."""@en ;
                                           dcterms:license "https://creativecommons.org/licenses/by/4.0/legalcode" ;
-                                          owl:versionInfo "0.0.1" .
+                                          owl:versionInfo "1.0.0-beta" .
```

Comment:
Good catch!

Reply:
The version is hardcoded so many places that it is almost worth to write a script for updating it...